test/parallel/test-tls-basic-validations.js

```diff
@@ -56,3 +56,17 @@ assert.throws(() => tls.createSecurePair({}),
   assert(buffer.equals(Buffer.from('abcd')));
   assert(out.NPNProtocols.equals(Buffer.from('efgh')));
 }
+
+{
+  const buffer = new Uint8Array([...Buffer.from('abcd')]);
```

Comment:
Why using the spread operator? If I'm not work passing a buffer to the `Uint8Array` constructor (`new Uint8Array(Buffer.from('abcd'))`) works.